subworkflows/local/profiling.nf

```diff
@@ -67,6 +67,14 @@ workflow PROFILING {
                     [meta + [id: "${meta.id}${meta.single_end ? '_se' : '_pe'}"], reads]
             }
             .combine(databases)
+            .filter { it ->
```

Comment:
Please use proper nameds declarations for readabilit, something like `meta, reads, db_meta, db` or something like that.

Reply:
Updated!